### PR TITLE
Set up alert stream access to invited users

### PR DESCRIPTION
**Pull request by dmitryduev**
New users invited to groups by admins get alert stream access by association.

### CODE REVIEW

File: skyportal/onboarding.py [hunk 1]
```diff
@@ -110,19 +120,45 @@ def setup_invited_user_permissions(strategy, uid, details, user, *args, **kwargs
...
         DBSession().add(Group(name=user.username, users=[user], single_user_group=True))
+        # Add stream access to single user group
+        single_user_group = (
+            DBSession().query(Group).filter(Group.name == user.username).first()
+        )
+        for stream_id in stream_ids:
+            DBSession.add(
+                GroupStream(group_id=single_user_group.id, stream_id=stream_id)
+            )
+            DBSession.add(StreamUser(stream_id=stream_id, user_id=user.id))
```
Comment (acrellin) [inline skyportal/onboarding.py:148]: Should this be outside of this if block? We want to do this even if the single user group might exist, right?

**Review by acrellin — COMMENTED** [review]
Sounds like we're still discussing some of the higher level stuff going on here, but will leave this one for now.

File: skyportal/onboarding.py (hunk 1 above)
Comment (dannygoldstein) [reply skyportal/onboarding.py:148]: Does this even need a block (i.e., do we need the `if` on 137)? How would the single user group exist already for an invited user?

**Review by dannygoldstein — COMMENTED** [review]
see rider to ari's comment

File: skyportal/onboarding.py (hunk 1 above)
Comment (acrellin) [reply skyportal/onboarding.py:148]: yeah, that can probably be removed. It was an extra precaution during the development phase, but I think the checks prior to this point are robust enough

Comment (dmitryduev) [reply skyportal/onboarding.py:148]: Ok, removed the check

**Review by dannygoldstein — APPROVED** [review]

File: skyportal/onboarding.py [hunk 2]
```diff
@@ -110,19 +120,42 @@ def setup_invited_user_permissions(strategy, uid, details, user, *args, **kwargs
...
+            .filter(GroupStream.group_id == group_id)
+            .all()
+        )
+        for stream in streams:
+            stream_ids.add(stream.id)
+
+    # Create single-user group
+    DBSession().add(Group(name=user.username, users=[user], single_user_group=True))
+    # Add stream access to single user group
+    single_user_group = (
```
Comment (acrellin) [inline skyportal/onboarding.py:140]: Minor thing that can wait til later -- we can just assign this to the group instance being added above, and won't need to re-fetch it here

**Review by acrellin — APPROVED** [review]
Looks good

**Pull request merged**